src/nebula-hooks/use-announce-and-translations.ts

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect } from '@nebula.js/stardust';
+import registerLocale from '../locale/src';
+import { Translator } from '../types';
+
+// eslint-disable-next-line no-shadow
+enum AnnouncerElements {
+  FIRST = 'first-announcer-element',
+  SECOND = 'second-announcer-element',
+}
+
+type AnnounceArgs = {
+  keys: Array<string | Array<string>>;
+  shouldBeAtomic?: boolean;
+  politeness?: 'polite' | 'assertive' | 'off';
+};
+
+/* creates the function for announcement */
+export const announcementFactory = (rootElement: Element, translator: Translator, prevAnnounceEl?: string) => {
```

Comment:
@haxxmaxx I just realized that prevAnnounceEl never gets a value which does not look correct.

Reply:
from a typing perspective?